### DISCUSSION TITLE
Mock reader: Trigger errors on demand

### DISCUSSION THREAD

**Discussion by inactivist** [TimOrme/aqimon#29] [Ideas]
I'm thinking it would be a nice-to-have addition if we could trigger errors in the mock reader via a side-channel API or via env variables (for example) and in the default configuration disable the default mode of generating random errors.

One immediate benefit (for me) would be to de-clutter the console output when doing development for the "happy path".  (OOB developer experience seems sub-optimal.)

**Comment by TimOrme** [comment]
The error rate is already configurable at the mock reader level itself: https://github.com/TimOrme/aqimon/blob/main/aqimon/read/mock.py#L14, so to implement it as an env var would just mean adding that to config.py.

I would vote for that over an API end point, just since it feels a little less "productized" to me.
